gateway/test/services/evm.tx-storage.test.ts

```diff
@@ -0,0 +1,94 @@
+import { EvmTxStorage } from '../../src/services/evm.tx-storage';
+import 'jest-extended';
+
+describe('Test local-storage', () => {
+  it('save, get and delete a key value pair in the local db', async () => {
+    const testChain1 = 'ethereum';
+    const testChain1Id = 423;
+    const testChain1Tx1 =
+      '0xadaef9c4540192e45c991ffe6f12cc86be9c07b80b43487e5778d95c964405c7';
+    const testChain1GasPrice1 = 200000;
+    const testChain1Tx2 =
+      '0xadaef9c4540192e45c991ffe6f12cc86be9c07b80b43487edddddddddddddddd';
+    const testChain1GasPrice2 = 200300;
+    const dbPath = '/tmp/ethereum.tx-storage.test.level';
```

Comment:
Please change to the `fs.mkdtemp()` and `fs.remove()` way used in `config-manager-v2.test.ts` - make sure the temp path is really yours and that it's cleaned up properly after tests.